README.rst

```diff
@@ -27,15 +27,29 @@ And an Array can be converted to a numpy ndarray on demand::
     array([ 0.72151309,  0.54654914], dtype=float32)
 
 
-Get in touch!
--------------
+------------------------------------------
+Important Note : Future development status
+------------------------------------------
+Biggus was originally developed primarily to provide large data handling for
+`Iris <http://github.com/SciTools/iris>`_ .
+
+For most applications it has now been surpassed by
```

Comment:
> masked data handling is still much easier with Biggus

True! But only for the moment... And I'm definitely happier with this _not_ having that extra level of specificity for mentioning masks directly.